pandas/_libs/tslibs/ccalendar.pyx

```diff
@@ -27,7 +27,7 @@ cdef int* sakamoto_arr = [0, 3, 2, 5, 0, 3, 5, 1, 4, 6, 2, 4]
 # The first 13 entries give the month days elapsed as of the first of month N
 # (or the total number of days in the year for N=13) in non-leap years.
 # The remaining 13 entries give the days elapsed in leap years.
-cdef int32_t* _month_offset = [
+cdef int32_t* month_offset = [
```

Comment:
Maybe an optimization for the future. But we can maybe remove half the values and have the cals just do:

`month_offset[month_index] + int(month > 1)`